templates/web/default/archive-old-enquiries.txt

```diff
@@ -0,0 +1,7 @@
+FixMyStreet is being updated in [[Council_name]] to improve how problems get reported.
```

Comment:
We don't have a default web directory, I don't think we want to create one for this. I'd say under `data` would be best place for this, if not just directly in the script itself.